runtime/near-vm-logic/src/logic.rs

```diff
@@ -2203,6 +2203,7 @@ impl<'a> VMLogic<'a> {
 
 #[derive(Debug, PartialEq, Serialize, Deserialize, Clone)]
 pub struct VMOutcome {
+    #[serde(with = "crate::serde_with::u128_dec_format")]
```

Comment:
Why not use `u128_dec_format_compatible` everywhere?

Reply:
Actually VMOutcome only need `Serialize`